submissions/Harnytskyi/task_js_dom/js/script.js

```diff
@@ -0,0 +1,96 @@
+        img: "img/crying-cat.jpg"
+    },
+    {
+        title: "Whoosh",
+        about: "Whoosht, known in Russia as вжух, is an image of a cat crudely dressed as a wizard that casts unfortunate spells. It grew popular on Russian websites such as Numl in December of 2016. The image of the cat is taken from a post on justcuteanimals.com (shown below, left). It later gained some traction as \"Wizard Cat\" on me_irl following a post from April 2016 that referred to it as the High Wizard Cat (shown below, right).",
+        img: "img/whoosh-cat.jpg"
+    }
+]
+
+const nav_ul = document.querySelector(".nav_ul");
+const aside = document.querySelector("aside");
+const main = document.querySelector("main");
+const openbtn = document.querySelector(".nav-open-button");
+let navButtonListArray;
+
+function createNavButtons() {
+    const navButtonList = document.createDocumentFragment();
+
+    nav_ul.innerHTML = "";
+    memes.forEach(function (element) {
+        const navLi = document.createElement("li");
+        const navButtonItem = document.createElement("button");
+        navButtonItem.classList.add("nav-button");
+        navButtonItem.innerHTML = element.title;
+        navLi.append(navButtonItem);
+        navButtonList.append(navLi);
+    });
+    nav_ul.append(navButtonList);
+    navButtonListArray = Array.from(document.querySelectorAll(".nav-button"));
+}
+
+function displayMain({ target }) {
+    if (target.classList.contains('nav-button')) {
+        highlightSelectedItem(target);
+    }
+    const selectedButtonIndex = navButtonListArray.indexOf(target);
+    main.innerHTML = "";
+    const memeInfo = document.createDocumentFragment();
+    let memeInfoitem;
+
+    const memeInfoTitle = document.createElement("div");
+    memeInfoTitle.classList.add("mainItem-title");
+    memeInfoTitle.innerHTML = memes[selectedButtonIndex].title;
+    memeInfo.append(memeInfoTitle);
+    const memeInfoImage = document.createElement("img");
+    memeInfoImage.classList.add("mainItem-image");
+    memeInfoImage.src = memes[selectedButtonIndex].img;
+    memeInfo.append(memeInfoImage);
+    const memeInfoAbout = document.createElement("div");
+    memeInfoAbout.classList.add("mainItem-about");
+    memeInfoAbout.innerHTML = "About:<br>" + memes[selectedButtonIndex].about;
+    memeInfo.append(memeInfoAbout);
+    main.append(memeInfo);
+}
+function hideMenu(target) {
+    aside.classList.toggle('hidden');
+}
+
+function highlightSelectedItem(target) {
+    if (document.querySelector('.nav-button.highlight') !== null)
```

Comment:
You can create a variable for storing the `document.querySelector('.nav-button.highlight')` in order to prevent to query your DOM node two times.
Also, you can check this node for being not falsy. More info about truthy and falsy values you can find [here](https://www.sitepoint.com/javascript-truthy-falsy/)